src/screens/AddNewItemScreen/index.tsx

```diff
@@ -64,9 +89,24 @@ export const AddNewItemScreen: FC = () => {
     });
   }, [onPressBack, navigation]);
 
-  const onPress = () => {
+  const onPressAdd = () => {
     addItem(text);
-    navigation.goBack();
+  };
+
+  const onPressEdit = () => {
+    console.log('hello');
+    stopNotificationHelper(editItem.notificationId);
+    const newItem = createNewItemHelper(text, date);
+    createNotificationHelper({ newItem, date, channelId });
+    dispatch(editItemSagaAction({ id: editItem.id, text, callback }));
```

Comment:
you still don't support reminder date in your edit logic!

Reply:
i support edit notification time

Reply:
i fix it